changelog/1054.doc.rst

```diff
@@ -0,0 +1 @@
+Created an example notebook demonstrating how the `SyntheticProtonRadiograph` class can be used to generate synthetic proton radiographs with arbitrary source profiles.
```

Comment:
```suggestion
Created an example notebook demonstrating how the
`~plasmapy.diagnostics.proton_radiography.SyntheticProtonRadiograph` class can be
used to generate synthetic proton radiographs with arbitrary source profiles.  Add
code documentation links to all proton radiograph notebooks.
```

This will allow `intersphinx` to properly generate the link for `SyntheticProtonRadioraph`.